publish-docs.js

```diff
@@ -125,31 +125,21 @@ function checkOutBranch() {
 
 function build() {
     try {
-        const options = {
-            files: ['doczrc.js'],
-            from: /base: '\/'/g,
-            to: `base: '/versions/${version}/'`,
-        };
-        replace.sync(options);
-
-        const options2 = {
-            files: ['src/examples/example.tsx', 'src/examples/props.tsx'],
-            from: /const BASE_URL = '\/';/g,
-            to: `const BASE_URL = '/versions/${version}/';`,
-        };
-        replace.sync(options2);
-
         const options3 = {
```

Comment:
I guess we can update the `options3` and `options4`. Maybe `indexReplaceOptions` and `stencilConfigReplaceOptions`?

Reply:
(And yes, I'm the one guilty of naming them `optionX` in the first place 😆)

Reply:
We could also just have `options` and reuse that variable for the different cases

Reply:
:zap: